---
bool success   # indicate successful run of triggered service
object_recognition_msgs/RecognizedObjectArray objects  # Recognized objects
